### ISSUE TITLE
Inconsistent active states across the editor UI

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#41643]
## What problem does this address?

Was reviewing the current UI and noticed a few inconsistencies across active states for controls. We have three standard states for indicating an active state, two shades of black and the admin primary color. 

#### Uses `#2f2f2f`/ ($gray-800):

<img width="278" alt="CleanShot 2022-06-09 at 13 39 59@2x" src="https://user-images.githubusercontent.com/1813435/172910071-2bbf2cf2-2a55-4b85-83fd-a7ca618d5ee0.png">

#### Uses `#1e1e1e`/ ($gray-900):

<img width="343" alt="CleanShot 2022-06-09 at 13 38 29@2x" src="https://user-images.githubusercontent.com/1813435/172909805-d39c1aa4-5168-47bb-94bf-8bb38c1f534a.png">
<img width="276" alt="CleanShot 2022-06-09 at 13 39 12@2x" src="https://user-images.githubusercontent.com/1813435/172909917-dffbe6c1-6186-4568-b2d9-f052af33ae16.png">
<img width="270" alt="CleanShot 2022-06-09 at 13 42 05@2x" src="https://user-images.githubusercontent.com/1813435/172910435-dd213437-7c61-47d2-a7a2-f490a95ead9f.png">

#### Uses `var(--wp-admin-theme-color)`:

<img width="271" alt="CleanShot 2022-06-09 at 13 40 32@2x" src="https://user-images.githubusercontent.com/1813435/172910157-348c8b55-f7b1-4c2d-96f5-aa83d189c3ea.png">
<img width="348" alt="CleanShot 2022-06-09 at 13 46 33@2x" src="https://user-images.githubusercontent.com/1813435/172911193-84759efc-f0d3-42f8-a3a1-daaf4d4ba871.png">

## What is your proposed solution?
I propose we choose one of the shades of black (either `#2f2f2f` ($gray-800)  or `#1e1e1e` ($gray-900)). Then employ that color consistently across all active states (including where we're currently using the primary admin color for the link/unlink UI).

**Comment by jameskoster** [comment]
Agreed, with perhaps the exception of List View. There the theme color creates a visual link with the canvas that can be helpful, especially when we get around to https://github.com/WordPress/gutenberg/issues/32163.

Should we also consider toggles, checkboxes, radios? Maybe tokenfields too?

<img width="298" alt="Screenshot 2022-06-10 at 10 27 59" src="https://user-images.githubusercontent.com/846565/173035936-03f0e97a-594c-457c-8d73-4caf4a68b0cd.png">
<img width="201" alt="Screenshot 2022-06-10 at 10 28 06" src="https://user-images.githubusercontent.com/846565/173035944-ddbd4fa1-21af-48af-838a-d156a028c6b8.png">
<img width="287" alt="Screenshot 2022-06-10 at 10 28 35" src="https://user-images.githubusercontent.com/846565/173035946-739500f4-5a20-42ac-939f-aac314f3e09f.png">


Edit:

A couple of other considerations in the datepicker:

<img width="307" alt="Screenshot 2022-06-10 at 10 37 11" src="https://user-images.githubusercontent.com/846565/173037455-866b47e3-149f-4fa5-892f-3d5a270979a3.png">

**Comment by richtabor** [comment]
> Agreed, with perhaps the exception of List View. There the theme color creates a visual link with the canvas that can be helpful, especially when we get around to https://github.com/WordPress/gutenberg/issues/32163.

That's the one I'm hesitant on most as well.
